windows/Dockerfile

```diff
@@ -25,3 +25,4 @@ FROM ${BASE_IMAGE}
 COPY --from=build /bin/ryuk /bin/ryuk
 CMD ["/bin/ryuk"]
 LABEL org.testcontainers.ryuk=true
+HEALTHCHECK --start-period=5s --interval=2s --timeout=2s --retries=10 CMD netstat -ltn | grep -q ":8080"
```

Comment:
I do not think the `CMD` will work on Windows (nanoserver).

Reply:
@HofmeisterAn 

Since I'm not on windows, and am not familiar with nanoserver, I thought it best to revert the changes on `windows/Dockerfile`.  It would be best for someone with Windows experience to implement and test a Windows health check.